.github/workflows/cd.yml

```
@@ -0,0 +1,31 @@
+    branches: [main, develop, cd-testing, cd-settings]
+  pull_request:
+    branches: [main, develop, cd-testing, cd-settings]
+
+jobs:
+  deploy:
+    if: github.event.pull_request.merged == true || github.event_name == 'push'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)